pkg/graphviz/traceflow_test.go

```diff
@@ -369,3 +600,11 @@ func TestGetTraceflowStatusMessage(t *testing.T) {
 		})
 	}
 }
+
+func TestFindClusterString(t *testing.T) {
+	graphStr := ""
+	clusterName := "cluster_source"
+	_, gotEndIndex := findClusterString(graphStr, clusterName)
```

Comment:
why are you skipping the startindex, and not asserting it's value, any specific reasons.